### PR TITLE
smart-qa 0.0.2

### PR DESCRIPTION
**Pull request by kot10n5tn**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/smart-qa/0.0.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- test
+toolkit:
+  go:
+    package_name: github.com/Levetty/bitrise-step-smartqa
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_path: null
```
Comment (koral--) [inline steps/smart-qa/0.0.2/step.yml:28]: It should default to `$BITRISE_APP_DIR_PATH` or something like that perhaps.

File: steps/smart-qa/0.0.2/step.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  opts:
+    description: |
+      Path to your .app file.
```
Comment (koral--) [inline steps/smart-qa/0.0.2/step.yml:31]: .app is a directory not file AFAIK

**Comment by kot10n5tn** [comment]
Thank you for your reviewing. I have updated step.yml.

**Review by zsolt-marta-bitrise — APPROVED** [review]

**Comment by zsolt-marta-bitrise** [comment]
@kot10n5tn Please sign the CLA before we can accept this change. You can find the link in the [comment](https://github.com/bitrise-io/bitrise-steplib/pull/3064#issuecomment-850788765) made by the bot. Thanks!

**Comment by kot10n5tn** [comment]
Thanks. I have accepted the CLA.

**Comment by kot10n5tn** [comment]
I should have already agreed, is there anything else I should do?image

**Comment by zsolt-marta-bitrise** [comment]
Thanks! Somehow the bot didn't pick up the change, I've merged it anyway

**Pull request merged**